_sections/act_and_prepare/en/02-do/19-shop-when-at-risk.md

```diff
@@ -1,9 +1,9 @@
-{:.only_toc}
+{:.do}
 ### Shopping when at risk
 
-{:.do.no_toc}
```

Comment:
The removal of these classes is causing a bit of a regression. You'll notice in the preview that the three main section headings no longer match.

Reply:
I don't care much about stripping the styles from "Things you should do" or "Things you should not do" but the same should happen for "Myths" if we do that. And, ideally, we'd remove the unused CSS rules that will be left over.

Reply:
Oh. I thought that when both {:.only_toc} and {:.do.no_toc} had the same header they could be collapsed into one. At least that was the way it used to be in the older version of the website.

Do we have some documentation or summary of how these mdfiles should be structured that I can look at so that I avoid making these erroneous assumptions?